<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createaccount</class>
 <widget class="QDialog" name="createaccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>457</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>创建账户</string>
  </property>
  <widget class="QLineEdit" name="login_passwdInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="login_backButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_accountInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777211</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="login_passwdLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_okButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_accountLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_confirmPasswdLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_confirmPasswdInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="login_nameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_nameInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="login_addressLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_addressInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="login_typeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>储种</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_interestLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>利息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_interestInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string>1.98</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="login_employeeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>员工号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_employeeInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="login_repeatPasswdState">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="login_typeCombobox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>一年期</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三年期</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>五年期</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
